tools/nightly-unit-tests/generate_bitrise_yml.py

```diff
@@ -76,9 +76,8 @@ def generate_bitrise_yml(test_plan: TestPlan, dry_run: bool):
     else:
         for simulator in missing_simulators:
             start = time.time()
-            print(f' → xcversion simulators --install="{simulator.os_name} {simulator.os_version}"')
             if not dry_run:
-                print(shell_output(f'xcversion simulators --install="{simulator.os_name} {simulator.os_version}"'))
+                print(shell_output(f'sudo xcodes runtimes install "{simulator.os_name} {simulator.os_version}"'))
```

Comment:
question/ No problem with `sudo` on CI 🤔? Is `sudo` required?

Reply:
yes, sudo is required. I was surprised too how did it work in bitrise but apparently it worked.